CHANGES.md

```diff
@@ -32,6 +32,10 @@ To be released.
  -  Bumped `BlockMetadata.CurrentProtocolVersion` to 4.  [[#PBFT]]
  -  Changed `IPreEvaluationBlockHeader.PreEvaluationHash` type from
     `ImmutableArray<byte>` to `HashDigest<SHA256>`.  [[#PBFT]]
+ -  Added `IStore.GetLastCommit(long height)` method.  [[#PBFT]]
+ -  Added `IStore.PutLastCommit(BlockCommit lastCommit)` method.  [[#PBFT]]
+ -  Added `IStore.DeleteLastCommit(long height)` method.  [[#PBFT]]
+ -  Added `IStore.GetLastCommitIndices()` method.  [[#PBFT]]
```

Comment:
```suggestion
 -  Added `IStore.GetLastCommit(long)` method.  [[#PBFT]]
 -  Added `IStore.PutLastCommit(BlockCommit)` method.  [[#PBFT]]
 -  Added `IStore.DeleteLastCommit(long)` method.  [[#PBFT]]
 -  Added `IStore.GetLastCommitIndices()` method.  [[#PBFT]]
```
We used to skip parameter name in changelog.

Reply:
Amended! https://github.com/planetarium/libplanet/pull/2348/commits/4805c8a868eaca9c2224548552f26f9f1d07b180